Themes/default/languages/LightPortal/.spanish_es.php

```diff
@@ -175,13 +198,20 @@
 $txt['lp_page_description'] = 'Descripción';
 $txt['lp_page_keywords'] = 'Palabras claves';
 $txt['lp_page_keywords_after'] = 'Usa una coma para separar';
-$txt['lp_permissions'] = array('Mostrar a los administradores', 'Mostrar a los invitados', 'Mostrar a los usuarios', 'Mostrar a todos');
+$txt['lp_page_publish_datetime'] = 'Fecha y hora de publicación';
 
 $txt['lp_page_options'] = array(
 	'show_author_and_date' => 'Mostrar el autor y la fecha de creación',
 	'allow_comments' => 'Permitir comentarios'
 );
 
+// Tabs
+$txt['lp_tab_content'] = 'Contenido';
+$txt['lp_tab_seo'] = 'SEO';
+$txt['lp_tab_access_placement'] = 'El acceso y la colocación';
+$txt['lp_tab_appearance'] = 'Apariencia';
+$txt['lp_tab_tuning'] = 'Tuning'; // I don't understand the context
```

Comment:
The tuning tab contains specific settings of the current block.

Reply:
Great gives me an idea how to put it, thanks!